src/const.js

```diff
@@ -0,0 +1,21 @@
+const pageBody = document.querySelector('body');
+
+const extraContainerTitles = {
+  TOP_RATED: 'Top rated',
+  MOST_COMMENTED: 'Most commented'
+};
```

Comment:
С заглавной перечисление
```suggestion
const ExtraContainerTitles = {
  TOP_RATED: 'Top rated',
  MOST_COMMENTED: 'Most commented'
};
```